### ISSUE TITLE
What modifications can be done to optimally combine ASL signal instead of BOLD

### ISSUE THREAD

**Issue by sboylan** [ME-ICA/tedana#1046]
### Summary
Modify/fork tedana to include ASL time-series.

**Comment by tsalo** [comment]
I am not familiar with multi-echo ASL generally. The only paper I'm aware of that discusses it is [Mahroo et al. (2021)](https://www.frontiersin.org/journals/neuroscience/articles/10.3389/fnins.2021.719676/full). [ExploreASL](https://sites.google.com/view/exploreasl/projects?authuser=0) does support multi-echo ASL for blood-brain barrier perfusion, but I don't know much about it.

If you could share more details about how the ASL (control, label, M0) signal curves would look and what, if any, differential effect you'd expect to see between them, that would be very helpful for understanding whether tedana could help.

As an aside, I have recently started maintaining [ASLPrep](https://github.com/PennLINC/aslprep), so multi-echo ASL is particularly interesting to me.

**Comment by CesarCaballeroGaudes** [comment]
Certainly multi-echo ASL for BBB perfusion is very relevant!!

**Comment by CesarCaballeroGaudes** [comment]
Yes, I am aware of that sequence. @smoia and I have already collaborated with the developers of the multi-echo ASL-BOLD sequence that you mentioned, and a joint paper is under review. 
Similar to you, we are also working on developing new ASL + multi-echo BOLD for calibrated fMRI purposes. However, I don't see how combining multiple-echoes can help with the ASL signal. Then, what would the difference be with respect with the multi-echo BOLD? To the best of my understanding, the first echo is used for perfusion, whereas the later echoes are combined for BOLD contrast.

**Comment by sboylan** [comment]
Thanks for the input, there are probably some aspects that are not clear to me. 
Wouldn't the 2-4 echos have some information on the perfusion ? 
If the signal is noisy, wouldn't it be interesting to use different echos to get a better idea of the value of the decay slope of the signal to get a better measurement of the ASL signal which would give us a better measurement of CBF ?
I am probably wrong, but I don't fully understand why I guess.

**Comment by tsalo** [comment]
@sboylan sorry we never followed up. Did you ever find a satisfactory answer to your question?

**Comment by sboylan** [comment]
Thanks Taylor, I haven't figured out how to do it. But I'm sure there must be a way !
